app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorRequestView.java

```diff
@@ -17,18 +17,16 @@
 import ch.epfl.favo.R;
 import ch.epfl.favo.favor.Favor;
 import ch.epfl.favo.favor.FavorUtil;
-import ch.epfl.favo.map.GpsTracker;
 import ch.epfl.favo.map.Locator;
 import ch.epfl.favo.util.DependencyFactory;
-import ch.epfl.favo.util.LocationManagerDependencyFactory;
-import ch.epfl.favo.view.tabs.BottomDestinationTab;
-
-/**
- * allows user to create favor objects that can be posted in the DB
- */
-public class FavorRequestView extends BottomDestinationTab implements View.OnClickListener {
-
+import ch.epfl.favo.view.ViewController;
 
+/** allows user to create favor objects that can be posted in the DB */
+public class FavorRequestView extends Fragment implements View.OnClickListener {
+  // TODO: Rename parameter arguments, choose names that match
+  // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+  private static final String ARG_PARAM1 = "param1";
```

Comment:
i think we can remove these two variables since they arent used, if we have to keep them we should consider renaming them  and removing the comment above it

Reply:
Agree, I'll remove it even though it's not my code

Reply:
Sorry changed my mind, I prefer to remove only things that are really not necessary and don't affect others. Since this is by someone else, I'll not touch it